sycl/test-e2e/ESIMD/regression/Inputs/dgetrf.hpp

```diff
@@ -47,7 +47,7 @@ using namespace sycl;
 using namespace std;
 using namespace sycl::ext::intel::esimd;
 
-ESIMD_PRIVATE ESIMD_REGISTER(192) simd<double, 3 * 32 * 4> GRF;
+ESIMD_PRIVATE ESIMD_REGISTER(256) simd<double, 3 * 32 * 4> GRF;
```

Comment:
@kychendev - I remember such issue was raised about a year ago - "Offset must be multiple to register size".

That time I wanted to add a comment/requirement to ESIMD SPEC telling the same - offset must be multiple to reg size, but you said it should not be a requirement and VC/BE or/and finalizer can support any offset.
Can you please confirm once again - was this test correct and if such offset must be supported by GPU BE?
+ @vmustya

Reply:
If offsets like 208 are supported, then we'll need to create a tracker for GPU RT to handle/fix that.